app/controllers/overrides/confirmations_controller.rb

```diff
@@ -10,8 +10,11 @@ def show
         # create client id
         add_resource_token
 
-        yield if block_given?
+        yield @resource if block_given?
```

Comment:
not 100% sure what this does but this is how it's done in the newer version of the gem. didn't actually seem to affect anything related to what we're using.